pkg/applicationmanager/controllers/applicationrestore.go

```diff
@@ -163,13 +163,81 @@ func (a *ApplicationRestoreController) verifyNamespaces(restore *storkapi.Applic
 func (a *ApplicationRestoreController) createNamespaces(backup *storkapi.ApplicationBackup,
 	backupLocation string,
 	restore *storkapi.ApplicationRestore) error {
+	logrus.Info("Starting createNamespaces")
 	var namespaces []*v1.Namespace
 
+	// rtTemplate := &storkapi.ResourceTransformation{}
+	// a.client.Get(context.Background(), types.NamespacedName{Name: restore.Spec.ResourceTransformationTemplate, Namespace: kdmputils.AdminNamespace}, rtTemplate)
+
+	var gvkResourceTransform map[schema.GroupVersionKind][]storkapi.TransformSpecs
```

Comment:
@sgajawada-px  The array nature of TransformSpecs will be effectively utilised only in future wherein we have multiple set of rules for a single type of resource and it is via a two different selector. As of current UX we don't support this via UI.

Reply:
Yes it's just good to have support from our end.